ggml-sycl.cpp

```diff
@@ -17287,7 +17260,6 @@ GGML_CALL static bool ggml_backend_sycl_supports_op(ggml_backend_t backend, cons
         case GGML_OP_CONT:
         case GGML_OP_DIAG_MASK_INF:
         case GGML_OP_SOFT_MAX:
-        case GGML_OP_ROPE:
```

Comment:
Why are you making this change, does this not mean we won't test the `ROPE` opeartor?

Reply:
No, see L17241 - we test ROPE if the input is contiguous.